packages/containers-shared/src/utils.ts

```diff
@@ -250,3 +243,88 @@ export async function checkExposedPorts(
 export function generateContainerBuildId() {
 	return randomUUID().slice(0, 8);
 }
+
+type DockerContext = {
+	Current: boolean;
+	Description: string;
+	DockerEndpoint: string;
+	Error: string;
+	Name: string;
+};
+
+/**
+ * Run `docker context ls` to get the Docker socket from the currently active Docker context
+ */
+export function getDockerSocketFromContext(dockerPath: string) {
```

Comment:
Maybe:
- add an explicit return type
- (and/or) add a JSDoc comment about the return value
- specify if it can throw in the JSDoc (i.e. when the socket is not found)